util/passportUtil.js

```diff
@@ -0,0 +1,61 @@
+const LocalStrategy = require("passport-local").Strategy
+const bcrypt = require("bcrypt")
+
+function initialize(passport) {
+    const authenticateUser = async (req, email, password, done) => {
```

Comment:
Why not `async function authenticateUser() ...`?

Reply:
Arrow function does not serve a purpose here, prob just pasted from the docs, right 😃